src/styles/components/stars.scss

```diff
@@ -0,0 +1,23 @@
+.stars {
+  width: 100px;
+  box-sizing: border-box;
+  display: flex;
+  justify-content: space-around;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    box-sizing: border-box;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: left;
```

Comment:
The `background-position: left;` will align the star image to the left of the container. If you want the stars to be centered within their respective containers, you should use `background-position: center;`. Ensure that this change aligns with the design requirements.